### PR TITLE
Dropdown component

### LINKED ISSUE #3: Create a Dropdown component for Filter
Problem
---
We need to create a component to display the the dropdown.

Do This
---
Create the Dropdown component that will display the the options from the "spreadsheet" so we can filter the cards on the main screen.

- Maybe Bootstrap might help out with this.
- Check if there is perhaps a prepackaged react component we can use for this.

Props we want to pass in 
---
- Background text (ex filter, current phase of position)

OnClick of the option we want to make sure that the selection does occur to the state as well.

Links
---
- [The actual figma to reference](https://www.figma.com/file/BtVrbVMiazxniE5FzXodcy/Internship-Log) 

Time Estimate
---
- 3-4 hours

### PR DESCRIPTION
**Pull request by rsmith92**
-Shown below is the dropdown button as it will be when not being used as well as what it looks like when you click on it.
-The labels for the button can easily be changed depending on what we want to use the dropdown for

image

image

### CODE REVIEW

**Review by mgray07 — COMMENTED** [review]
added props to dropdown, app

**Comment by mgray07** [comment]
done

**Comment by srirag-vuppala** [comment]
We still need to merge this in

File: react_frontend/src/components/Dropdown_Button.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <Dropdown.Item href="/">Coding Challenge</Dropdown.Item>
+        <Dropdown.Item href="/">Pre-recorded Interview</Dropdown.Item>
+        <Dropdown.Item href="/">In-person Interview</Dropdown.Item>
+        <Dropdown.Item href="/">Offer</Dropdown.Item>
+        </Dropdown.Menu>
+        </Dropdown>
+    )  
+}
+
+export default Dropdown_Button;
```
Comment (mgray07) [inline react_frontend/src/components/Dropdown_Button.js:34]: I added props, I think this will make it more versatile in the future, LGTM though

Comment (rsmith92) [reply react_frontend/src/components/Dropdown_Button.js:34]: Thank you, LGTM as well

File: react_frontend/src/assets/Dropdown_Button.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.Dropdown_Button[aria-expanded="true"] {
```
Comment (srirag-vuppala) [inline react_frontend/src/assets/Dropdown_Button.css:1]: LGTM but to override the actual bootstrap color config using the bsPrefix prop is necessary!

File: react_frontend/src/components/Dropdown_Button.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        {/* <Dropdown.Toggle class= "text-danger" variant="success" id="dropdown-basic"> */}
+        {props.FilterType}
+        </Dropdown.Toggle>
+        <link rel="stylesheet" type="text/css" href="Dropdown_Button.css"></link>
+        
+        
+
+
+        <Dropdown.Menu>
+        <Dropdown.Item href="/">Waiting to Hear Back</Dropdown.Item>
```
Comment (srirag-vuppala) [inline react_frontend/src/components/Dropdown_Button.js:24]: Should have removed the hrefs for usability since we aren't rerouting. LGTM though 👍🏾

**Comment by hhokari** [comment]
You might want to consider responsiveness of the dropdown component on different screen sizes for the future, looks good though.

**Pull request merged**